### ISSUE TITLE
*** Release Notifications    -    Comment to be updated of the latest releases

### ISSUE THREAD

**Issue by lacymorrow** [lacymorrow/crossover#136]
I need folks to test pre-release versions for regressions before I push anything live to automatically update.

Anyone interested? 

Comment here

**Comment by lacymorrow** [comment]
New Pre-release, v2.5.5: https://github.com/lacymorrow/crossover/tree/v2.5.5

- Add button to reset settings
- Add button to choose crosshair in settings
- Add button to choose custom crosshair (instead of just click/drag)

- Adding updates to electron-preferences
- Update Electron to v12
- Update dependencies

**Comment by lacymorrow** [comment]
Release 2.6 coming soon
- notifications
- sound
- dark/light mode
- auto-update improvements

**Comment by lacymorrow** [comment]
2.7.4 release was a bust, could use more folks to test.

**Comment by lacymorrow** [comment]
Pre-release v3: https://github.com/lacymorrow/crossover/releases/tag/v3.0.0

**Comment by Eccomi21** [comment]
I am interested in testing. let me know what you need/what features you need testing for

**Comment by Eccomi21** [comment]
@lacymorrow glad to help if I actually can help. You probably don't even know how much I value this app. Used to try and install reshade on everything to use nopeelz reshade xhair which half the time got blocked by some overprotective anticheat. And since I can't contribute code I can at least ~~complain~~ give feedback.

**Comment by lacymorrow** [comment]
Complaints are also welcome!

**Comment by Daytrip-Society** [comment]
This project has improved head-clickability for me by a non-scientific nine thousand percent, give or take one. Any way I can lab-rat myself or my setup in the pursuit of making this cooler is alright by me. 

Always running whatever the newest insider builds on Win11 are, and playing Fortnite (mostly) through the unofficial legendary-gl launcher, so I might catch some edge-cases of weirdness. 

If I get bored enough I might start making crosshairs to add to the library if submissions are open, as my only complaint thus far is that there aren't many that I'd call 'top-heavy'- think Ffp Cch, Crossbones, or a hypothetical inverted 151 or 152. Not sure how many people like that style, but it subconsciously helps me manage recoil somehow. 🤷 

Currently changing setup every other day, but was getting some serious mileage out of this one earlier today:

![download (1)](https://user-images.githubusercontent.com/99981846/166414810-ac59a9a7-069a-4451-802d-3245e321e6f8.png)

Thanks for the awesome project and the time you've devoted thus far :D

**Comment by Eccomi21** [comment]
@Daytrip-Society I made a crosshair out of Shreks head and even that works surprisngly well if you keep replacing the enemies head with shrek and start clicking.

![unknown](https://user-images.githubusercontent.com/55301370/166469130-bd722b69-b717-4b52-9cfd-86c651fcb8d5.png)

However i made my own GIF because i really liked the 6 color blinking one, but find it easier to hipfire close range if i have a longer vertical shape i can "draw" over the enemies body.

![cunthair](https://user-images.githubusercontent.com/55301370/166471172-43464c17-0f6b-4b4d-93d5-ce8c6249966d.gif)

I could imagine some people might find this annoying, but i literally never loose track of where my crosshair his at any given time becuse it keeps visually screaming at me, unlike apex' own shitty half translucent dynamic hipfire trashhair

**Comment by lacymorrow** [comment]
@Daytrip-Society I didn't know about `legendary-gl`, that's awesome!
I really only play Fortnite as well so I might consider trying it out in a Linux distro.

@Eccomi21 you guys are hilarious with your head-clicking. I received an email from someone who strictly uses the "Hello Kitty" crosshair for the same reason 😂 

Could you guys post your computer specs here? OS + version and CPU

If you guys create crosshairs, I'm happy to add them to the app. I created an issue specifically for it so just post any images here: https://github.com/lacymorrow/crossover/issues/178

If someone is feeling like doing some design work, I just added another ticket to convert the kenney crosshairs into SVG: https://github.com/lacymorrow/crossover/issues/179

Cheers fellas 🍻

**Comment by Eccomi21** [comment]
@lacymorrow 

OS: Windows 10 Kernel 10.0.19044
CPU: Intel i7-8700K (12) @ 5.000GHz

Yes i did switch back from arch, I admitted defeat. Interestingly enough it was way better for gaming, just everything else didn't work out in my favor. I wish I could've been an advocate for the Linux community here xD

**Comment by Daytrip-Society** [comment]
@lacymorrow 

Back again, had a fairly unfun series of events keep me from devoting time the past month or so, and notifications for this account got borked somehow on top of it. Fix attempted, we'll see how this goes 😅

As requested, system deets: 
Acer Aspire A515-51
OS: Windows 11 build 10.0.22622
CPU: Intel i5-8250U
GPU: Intel UHD 620

Upped to 20GB RAM, and with that plus an SSD swap it almost runs not like a potato sometimes.

**Comment by lacymorrow** [comment]
📋 Reviewed by Paperclip — categorized as discussion/community thread/wontfix. No task created.

**Comment by lacymorrow** [comment]
📋 Tracked in Paperclip as **LAC-1799**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by Eccomi21** [comment]
I don't know what you are conjuring up but whatever it is, if you need a tester i am still around.
Currently on Linux.

OS: NixOS 26.05 (Yarara) x86_64
WM: Hyprland 0.54.3 (Wayland)

**Comment by lacymorrow** [comment]
Tracked in Paperclip as **LAC-2587**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
Closing — this call for testers is from 2022. Testing for current releases happens through GitHub releases and the beta channel.
